sdk/communication/Azure.Communication.PhoneNumbers/samples/Sample2_PhoneNumberAdministrationClientAsync.md

```diff
@@ -1,7 +1,7 @@
 # Searching, reserving, purchasing, releasing phone numbers (Async)
 
 This sample demonstrates how to search, reserve, purchase and release phone numbers in Azure Communication Services.
-To get started, you'll need a URI to an Azure Communication Services. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.Administration/README.md) for links and instructions.
+To get started, you'll need a URI to an Azure Communication Services. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/communication/Azure.Communication.PhoneNumbers/README.md) for links and instructions.
 
 ## Creating a PhoneNumberAdministrationClient
```

Comment:
Would be nice to update everything that still says Administration

```suggestion
## Creating a PhoneNumbersClient
```

Reply:
The new swagger implementation does that renaming. I did not want to mix and cause conflict.